tests/visualize/test_viz_tree.py

```diff
@@ -0,0 +1,62 @@
+"""Tests the visualization functions for trees."""
+# _tree.py
+
+import pytest
+import jax.numpy as jnp
+import os
+from pathlib import Path
+
+
+from pyggdrasil.tree_inference._tree import Tree
+
+import pyggdrasil.visualize.tree as viz
+
+IN_GITHUB_ACTIONS = os.getenv("GITHUB_ACTIONS") == "true"
+
+
+@pytest.mark.skipif(IN_GITHUB_ACTIONS, reason="Test doesn't work in Github Actions.")
+def test_plot():
```

Comment:
I'd suggest to use [the `tmp_path` fixture](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html) to specify the path, rather than using `../data/trees/`. It's also possible (although I don't know how likely) that this was the issue with Github Actions (not all directories there are writeable, if I remember correctly)